roles/kubernetes-apps/external_provisioner/local_volume_provisioner/templates/local-volume-provisioner-ds.yml.j2

```diff
@@ -20,9 +20,10 @@ spec:
     spec:
       priorityClassName: {% if local_volume_provisioner_namespace == 'kube-system' %}system-node-critical{% else %}k8s-cluster-critical{% endif %}{{''}}
       serviceAccountName: local-volume-provisioner
+{% if local_volume_provisioner_tolerations %}
       tolerations:
-        - effect: NoSchedule
-          operator: Exists
+        {{ local_volume_provisioner_tolerations | to_nice_yaml(indent=2) | indent(width=8) }}
+{% endif %}
```

Comment:
This part had been added since the pull request https://github.com/kubernetes-sigs/kubespray/pull/2871
But I could not get the point of that.
In addition, the upstream manifest doesn't contain the tolerations as https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/b5987962ca8c6f53415c4ad683ddea4cb274fec6/helm/generated_examples/baremetal-namespace.yaml#L33
So it is fine to remove it for me by default.